docs/troubleshooting.md

```diff
@@ -2,60 +2,64 @@
 
 Having a little trouble with the MFTF? See some common errors and fixes below.
 
-## WebDriver issues
```

Comment:
This documentation change looks more complicated than it is.

I discussed with @dobooth and we agreed to remove the two existing troubleshooting items on this page because they were no longer relevant.